src/components/ProjectLogo/styles.less

```diff
@@ -0,0 +1,13 @@
+@import '../../assets/theme/color.less';
+
+.project-logo {
+  flex: none;
+  width: 80px;
+  height: 80px;
+  border-radius: 6px;
+  background: #070a26;
```

Comment:
here

Reply:
It will be improved in the next version.